src/FsAutoComplete.Core/Utils.fs

```diff
@@ -499,6 +499,38 @@ type ReadOnlySpanExtensions =
 
     line
 
+#if !NET7_0_OR_GREATER
+  [<Extension>]
+  static member IndexOfAnyExcept(span: ReadOnlySpan<char>, value0: char, value1: char) =
+    let mutable i = 0
+    let mutable found = false
+
+    while not found && i < span.Length do
+      let c = span[i]
+
+      if c <> value0 && c <> value1 then
+        found <- true
+      else
+        i <- i + 1
+
+    if found then i else -1
+
+  [<Extension>]
+  static member LastIndexOfAnyExcept(span: ReadOnlySpan<char>, value0: char, value1: char) =
+    let mutable i = span.Length - 1
+    let mutable found = false
+
+    while not found && i >= 0 do
+      let c = span[i]
+
+      if c <> value0 && c <> value1 then
+        found <- true
+      else
+        i <- i - 1
+
+    if found then i else -1
+#endif
```

Comment:
Thank you for adding these helpers!

Reply:
It feels a little odd adding only these specific overloads, but they're the only ones I needed...